oai_agents/common/tags.py

```diff
@@ -64,3 +68,20 @@ class TeammatesCollection:
     TRAIN = 'train'
     EVAL = 'eval'
 
+
+class LearnerType:
+    '''
+    Different learners would receive differnt rewards from the base_overcooked_env.py.
+    The primary difference between learners are on the weights of personal and group reward.
+    Saboteur and Selfisher weight personal reward positively and group reward negatively.
+    Soloworker, weight individual reward positively, and group reward with a zero.
+    Collaborator, and Helper weight both rewards positively.
+    If we grade how much they care about group: 
+    Helper > Collaborator > Soloworker >> Selfisher > Saboteur.
```

Comment:
We don't have to address this in the current PR: but I think we should come up with a better name for `Helper` in the future because the name does not imply an agent that has better collaboration than `Collaborator`

Reply:
How does Supporter sound like?

Reply:
I think BetterCollaborator might be better lol but no strong opinions